doc/releases/changelog-dev.md

```diff
@@ -153,6 +153,9 @@
   and the codecov check itself would never execute.
   [(#5101)](https://github.com/PennyLaneAI/pennylane/pull/5101)
 
+* `qml.Identity()` can be initialized without wires. Measuring it is currently not possible though.
```

Comment:
Does that include something like `qml.expval(qml.PauliX(0) @ qml.Identity())` or just strictly `qml.expval(qml.Identity())`?

Reply:
Good point, this case is in fact not handled. Though it _should_ also not appear accidentally since Identities are stripped from PauliWords upon initialization:
```
>>> pw = PauliWord({0:"X", 1:"I"})
>>> pw
X(0)
>>> pw.operation()
PauliX(wires=[0])
```
It _could_ occur with still if a user specifically tries to compute `qml.expval(qml.PauliX(0) @ qml.Identity())` though 😬  Luckily, a fix to handle empty-wire Identities is under way already anyway.